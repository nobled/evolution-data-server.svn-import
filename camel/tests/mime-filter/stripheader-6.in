Subject: this is a simple test
Stripped: of the header stripping filter
 ooh look this header spans many
	 lines look at all the freaking lines
 that this header spans. Wowwww.
Stripped: and then another one to strip that extends fairly far across the page
Other: let's not use CC here.
Stripped: oops, I stripped it again
Stripped: then single line....
   followed by multi-line. Huzzah.
NotBCC: definitely not BCC
  however this one is definitely continued
Stripped: god-damn look at that stripping.

This is the body of the message.