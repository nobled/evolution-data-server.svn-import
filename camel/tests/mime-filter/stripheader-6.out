Subject: this is a simple test
Other: let's not use CC here.
NotBCC: definitely not BCC
  however this one is definitely continued

This is the body of the message.